docs/troubleshooting.md

```diff
@@ -1,3 +1,49 @@
 # Troubleshooting Guide
 
-Content to be added.
+## Manual cleanup
+
+During Abort/Finalize LCA performs a cleanup. This includes:
+
+- Remove the old state root
+- Cleanup precaching resources
+- Delete OADP backups CRs
+- Remove IBU files from the file system
+
+If LCA failes to perform any of the above steps it transitions to Abort/Finalize failed state.
+The condition message and log shows which steps have failed.
+User should perform manual cleanup and add `lca.openshift.io/manualCleanupDone` annotation to IBU. After observing this annotation LCA will retry the cleanup and if successful, IBU transitions to `Idle`.
+
+### Stateroot cleanup
+
+During `Abort` LCA cleanups the new stateroot and during `Finalize` LCA cleanups the old stateroot. User should inspect the log to determine why this step has failed since failure in this step indicates serious issue in node.
+
+User can manually cleanups the stateroot using:
+
+```bash
+# specify the stateroot that should be cleaned
+# old stateroot for finalize and new stateroot for Abort
+stateroot="<>"
+
+function get_deployment_id {
```

Comment:
We should revise this. I think this is too complex for end-user troubleshooting section. Rather, I'd explain what's being done, for manual steps. ie:

- Run `ostree admin status` to determine if there are any existing deployments in the stateroot. These will need to be cleaned up first, using the `ostree admin undeploy` command.
- Once all corresponding deployments have been cleaned up, you can wipe the stateroot folder, with a command such as `blah blah blah`